src/core/load-fragment.c

```diff
@@ -2390,6 +2390,375 @@ int config_parse_documentation(const char *unit,
 }
 
 #ifdef HAVE_SECCOMP
+static int syscall_filter_parse_one(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                ExecContext *c,
+                bool invert,
+                const char *t,
+                bool warn) {
+        int r;
+
+        if (*t == '@') {
+                const char * const *s;
+                static const char * const syscall_sets[] = {
+                        /* Default list */
+                        "@Default",
+                        "execve\0"
+                        "exit\0"
+                        "exit_group\0"
+                        "rt_sigreturn\0"
+                        "sigreturn\0",
+
+                        /* Need superuser capabilities */
+                        "@SysAdmin",
```

Comment:
I don't quite follow, how would you get the same result by not listing?

Here's my polkitd dropin:
cat /etc/systemd/system/polkitd.service.d/local.conf 
[Service]
Capabilities=
CapabilityBoundingSet=
NoNewPrivileges=true
SecureBits=no-setuid-fixup no-setuid-fixup-locked keep-caps keep-caps-locked
PrivateNetwork=true
ProtectSystem=full
ProtectHome=true
PrivateTmp=true
PrivateDevices=true
DevicePolicy=closed
InaccessibleDirectories=-/dev/pts /dev/shm /dev/mqueue /setuid /boot
ReadOnlyDirectories=/
LimitNPROC=5
TasksMax=5
MountFlags=slave
SystemCallArchitectures=native
User=polkitd
Group=polkitd
SystemCallFilter=~@Special @Unimp @SysAdmin
SystemCallFilter=mount

This is more convenient than the resulting filter:
 systemctl show polkitd.service | grep Filter
SystemCallFilter=~_sysctl acct adjtimex afs_syscall bpf chown chroot create_module delete_module fchown fchownat finit_module get_kernel_syms get_mempolicy getpmsg init_module ioperm iopl kexec_file_load kexec_load lchown modify_ldt nfsservctl perf_event_open pivot_root putpmsg query_module quotactl reboot seccomp security setdomainname setfsgid setfsuid setgid setgroups sethostname setregid setresgid setresuid setreuid settimeofday setuid swapoff swapon syslog tuxcall umount2 uselib vhangup vserver